File: Easy/Split Array into Minimum Subsets/split-array-into-minimum-subsets.cpp
class Solution {
  public:
    int minSubsets(vector<int>& arr) {
        // code here
        sort(begin(arr) , end(arr));
        int count = 1;
        for(int i = 1 ; i < arr.size() ; i++){
            if (arr[i] - arr[i-1] != 1){
                count++;
            }
        }
        
        return count;
        
    }
};

File: Easy/Split Array into Minimum Subsets/split-array-into-minimum-subsets.py
class Solution:
    def minSubsets(self, arr):
        # Sort the array so that consecutive elements
        # are placed next to each other.
        arr.sort()

        # At least one subset is always required.
        count = 1

        # Traverse the sorted array.
        for i in range(1, len(arr)):

            # If the current element is not consecutive
            # to the previous element (difference != 1),
            # start a new subset.
            if arr[i] - arr[i - 1] != 1:
                count += 1

        # Return the minimum number of subsets.
        return count